framework/doc/content/source/auxkernels/MaterialRateRealAux.md

```diff
@@ -0,0 +1,26 @@
+# MaterialRateRealAux
+
+!syntax description /AuxKernels/MaterialRateRealAux
+
+## Description
+
+The `MaterialRateRealAux` AuxKernel is used to output the rate of change of
+material properties as an element-level, constant variable.
```

Comment:
doesn't have to be constant. could be a higher order monomial

Reply:
fixed